README.md

```diff
@@ -1,10 +1,12 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue) [![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![StackOverflow](https://img.shields.io/badge/StackOverflow-questions-blueviolet)](https://stackoverflow.com/questions/tagged/fairlearn)
 
-# Fairlearn
+# Fairlearn - Alpha release
```

Comment:
I'm not really sure how to feel about this one. Do you find this helpful to users?

I imagined what effect this would have on me if I saw this on another project. I think it would make it less likely for me to use it. We already list the alpha status in `setup.py`, so that makes me lean towards not adding it here as well. I know interpret does this, but we don't need to be copying from them. In terms of documentation we're probably ahead of that project (even though ours leaves A LOT to be desired right now as you know).

Reply:
@romanlutz Yes, I think it is important to communicate the maturity of a project :)